SlackPirate.py

```diff
@@ -153,12 +153,17 @@ def display_cookie_tokens(cookie):
         r = requests.get("https://slackpirate-donotuse.slack.com", cookies=cookie)
         already_signed_in_match = re.findall(ALREADY_SIGNED_IN_TEAM_REGEX, str(r.content))
         if already_signed_in_match:
+            selected_agent = getUserAgent()
             print(termcolor.colored("This cookie has access to the following Workspaces: \n", "white", "on_blue"))
             for workspace in already_signed_in_match:
                 r = requests.get(workspace, cookies=cookie)
                 regex_tokens = re.findall(SLACK_API_TOKEN_REGEX, str(r.content))
                 for slack_token in regex_tokens:
-                    print(termcolor.colored("URL: " + workspace + " Token: " + slack_token, "white", "on_green"))
+                    collected_output_info = init_scanning_context(token=slack_token, user_agent=selected_agent)
+                    if check_if_admin_token(token=slack_token, output_info=collected_output_info):
+                        print(termcolor.colored("URL: " + workspace + " Token: " + slack_token + ' (admin token!)', "white", "on_green"))
```

Comment:
I'd change the colour here to magenta so that an admin token is easily distinguishable from others. Also the colour will be consistent with the --token check.

Reply:
Agree. Pushed the changes to make it magenta.